Teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher
```

Comment:
- [ ] Kindly make sure that teacher inherits from person class like
`class Teacher < Person`
- [ ] Kindly make sure that parent_permission is added for initialize and super constructors.

Reply:
hey @sja-thedude 
_Thank you for review and changes were made 👍_